linux_os/guide/system/permissions/restrictions/poisoning/coreos_slub_debug_kernel_argument/rule.yml

```diff
@@ -23,7 +23,7 @@ identifiers:
 
 references:
     nist: CM-6(a)
-    srg: SRG-OS-000433-GPOS-00192
+    srg: SRG-APP-000243-CTR-000600,SRG-APP-000243-CTR-000600
```

Comment:
The SRG is duplicated here.
And should the `SRG-OS` be removed?

Reply:
Yes, the extra SRG-APP is not supposed to be there. The SRG-OS should be removed as the rule is not present in any control under `srg_gpos`.